src/middleware/isFlaggedAndIsAdmin.js

```diff
@@ -0,0 +1,8 @@
+const isAdmin = require("./isAdmin")
+
+module.exports = (req, res, next) => {
+	if (req.query.isFlagged) {
```

Comment:
for this, we should check for if the `isFlagged` query param is not defined. whether false or true

because it is a `boolean`, what we currently have will only run for when `isFlagged` is actually set to `true`.

Reply:
alright